monkey/monkey_island/cc/ui/src/services/configuration/polymorphism.js

```diff
@@ -0,0 +1,14 @@
+const POLYMORPHISM_SCHEMA = {
+  'title': 'Polymorphism',
+  'properties': {
+    'randomized_agent_hash': {
+      'title': 'Randomize Agent hash',
+      'type': 'boolean',
+      'default': false,
+      'description': 'Emulate the property of polymorphic (or metamorphic) malware that all ' +
+                     'copies have unique hashes by modifying the Agent binary before propagation.'
+    }
+  }
```

Comment:
I'm not sure how much of this is necessary at all. I've removed this locally, rebuilt, and seen no change in behavior. I've also done this for other parts of the configuration.

Most of the schema should be built from the config schema generated by the backend.